cloudbuild.yaml

```diff
@@ -0,0 +1,4 @@
+steps:
```

Comment:
I would actually split this out into its own git branch/PR, because it's mostly unrelated to the rest of the changes here.

Reply:
+1 to this suggestion

Reply:
will do! got exited to quickly implement it and didn't see its in the same PR